acm_certificate/main.tf

```diff
@@ -26,55 +26,53 @@ variable "validation_cname_ttl" {
 
 output "cert_arn" {
   description = "ARN of the issued ACM certificate"
-  value = "${element(concat(aws_acm_certificate.main.*.arn, list("")), 0)}"
+  value       = element(concat(aws_acm_certificate.main.*.arn, [""]), 0)
 }
 
 output "finished_id" {
   description = "Reference this output in order to depend on validation being complete."
-  value = "${element(concat(aws_acm_certificate_validation.main.*.id, list("")), 0)}"
+  value       = element(concat(aws_acm_certificate_validation.main.*.id, [""]), 0)
 }
 
-
 # -- Resources --
 
-
 # Create the certificate with the specified SubjectAltNames
 resource "aws_acm_certificate" "main" {
-  count = "${var.enabled}"
+  count = var.enabled
 
-  domain_name               = "${var.domain_name}"
-  subject_alternative_names = "${var.subject_alternative_names}"
+  domain_name               = var.domain_name
+  subject_alternative_names = var.subject_alternative_names
 
-  validation_method         = "DNS"
+  validation_method = "DNS"
 
   lifecycle {
     create_before_destroy = true
 
     # TODO: this is a workaround for an AWS API / Terraform AWS provider bug
     # https://github.com/terraform-providers/terraform-provider-aws/issues/8531
     # https://github.com/18F/identity-devops/issues/1469
-    ignore_changes = ["subject_alternative_names"]
+    ignore_changes = [subject_alternative_names]
   }
 }
 
 # Create each validation CNAME
 resource "aws_route53_record" "validation-cnames" {
-  count   = "${(length(var.subject_alternative_names) + 1) * var.enabled}"
-  name    = "${lookup(aws_acm_certificate.main.domain_validation_options[count.index], "resource_record_name")}"
-  type    = "${lookup(aws_acm_certificate.main.domain_validation_options[count.index], "resource_record_type")}"
-  zone_id = "${var.validation_zone_id}"
-  records = ["${lookup(aws_acm_certificate.main.domain_validation_options[count.index], "resource_record_value")}"]
-  ttl     = "${var.validation_cname_ttl}"
+  count = length(var.subject_alternative_names) + 1 * var.enabled
```

Comment:
We lost parens here and the logic is no longer correct.

Assuming `enabled` is turned into a boolean, it's really supposed to be this:
```hcl
count = var.enabled ? length(var.subject_alternative_names) + 1 : 0
```